external/CapnProto.cmake

```diff
@@ -21,23 +21,34 @@
 
 # Build Cap'n Proto from source.
 #
-# OUTPUTS
+# OUTPUT VARIABLES:
+#
+# CAPNP_STATIC_LIB_TARGET: name of static library target that contains all of
+#                          capnproto library objects.
 #
-# CAPNP_LINK_LIBRARIES: list of capnproto libraries (paths) needed by apps for linking
```

Comment:
In a future PR, we could easily apply this technique to Yaml/YamlCpp as well as Apr/AprUtils pairs so that the rest of our cmake code would only need to be concerned with a single library target for each pair instead of being exposed to the inter-dependencies within each pair.